install_config/persistent_storage/persistent_storage_nfs.adoc

```diff
@@ -11,124 +11,248 @@
-storage provider.
-====
-
-[[nfs-provisioning]]
+
+OpenShift clusters can be provisioned with
+link:../../architecture/additional_concepts/storage.html[persistent storage] using NFS
+Some familiarity with Openshift,
+link:../../architecture/additional_concepts/storage.html#persistent-volume-claims[persistent volume claims (PVCs)],
+and https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Storage_Administration_Guide/ch-nfs.html[NFS] is beneficial.
+
+The Openshift persistent volume (PV) framework allows administrators to provision a cluster
+with persistent storage and gives developers the ability to request those resources without
+having specific knowledge of the underlying infrastructure. Persistent volumes are not bound
+to a single project; they can be shared across the entire OpenShift cluster. Persistent volume
+claims (PVCs), however, are specific to a project and are created and used by developers.
+
+(*Note:* "project" and "namespace" are used interchangeably throughout this document)
 
 == Provisioning
-Storage must exist in the underlying infrastructure before it can be mounted as
-a volume in OpenShift. To provision NFS volumes in OpenShift, all that is
-required is a distinct list of NFS servers and paths and the
-`*PersistentVolume*` API.
 
-You must define your persistent volume in an object definition before creating
-it in OpenShift:
+Storage must exist in the underlying infrastructure before it can be mounted as a volume in
+OpenShift. To provision NFS volumes a list of NFS servers and export paths are all that is required.
 
-.Persistent Volume Object Definition Using NFS
-====
+*Note:*
+Each NFS volume must be mountable by all schedulable nodes in the cluster.
 
-[source,yaml]
-----
-apiVersion: "v1"
-kind: "PersistentVolume"
+A PV spec file is usually defined before creating the PV:
+```
+apiVersion: v1
+kind: PersistentVolume
 metadata:
-  name: "pv0001" <1>
+  name: pv0001 <1>
 spec:
   capacity:
-    storage: "5Gi" <2>
+    storage: 5Gi <2>
   accessModes:
-    - "ReadWriteOnce" <3>
+  - ReadWriteOnce <3>
   nfs: <4>
-    path: "/tmp" <5>
-    server: "172.17.0.2" <6>
-  persistentVolumeReclaimPolicy: "Recycle" <7>
-----
-<1> The name of the volume. This will be how it is identified via
-link:../../architecture/additional_concepts/storage.html[persistent volume
-claims] or from pods.
-<2> The amount of storage allocated to this volume.
-<3> This is the policy for restricting read/writes to a shared volume.
-<4> This defines the volume type being used, in this case the *nfs* plug-in.
-<5> The path that is exported by the NFS server.
-<6> The host name or IP address of the NFS server.
-<7> Defines what happens to a volume when released from its claim. Valid options
+    path: /tmp <5>
+    server: 172.17.0.2 <6>
+  persistentVolumeReclaimPolicy: Recycle <7>
+```
+<1> the name of the volume. This is the PV identify in various `oc ... pod` commands.
```

Comment:
s/the/The/

Need init caps on each of the beginnings of these callouts below as well.